data-raw/vs_metabolic.R

```diff
@@ -0,0 +1,295 @@
+# Dataset: vs_metabolic
+# Description: Create VS test SDTM dataset for metabolic studies
+
+# Load libraries ----
+
+library(tibble)
+library(dplyr)
+library(pharmaversesdtm)
+
+## Read input data ----
+
+data("vs")
+data("dm_metabolic")
+
+# Convert blank to NA ----
+vs <- convert_blanks_to_na(vs)
```

Comment:
Do we need this line? Isn't `pharmaversesdtm::vs` already in the right format? 

In any case`convert_blanks_to_na()` comes from `{admiral}` and so you'd need to add a `library(admiral)` call at the top of your script. But because this code will eventually be moved to `{pharmaversesdtm}`, which `{admiral}` depends on, this would create a sort of circular dependency.

As such I'd remove or find another way to do this step.

Reply:
Agreed. Removed it